### DISCUSSION TITLE
Do you open source Mojo

### DISCUSSION THREAD

**Discussion by akhildevelops** [modular/modular#76] [Q&A] [answered]
If you are planning to opensource the language, what licence are you planning for ?

**Comment by TZDak** [comment]
According to the web page at https://nothans.com/modular-the-future-of-ai-development-hello-mojo "The developers plan to progressively open-source Mojo over time", and I thought I had seen something similar somewhere on modular.com but I'm not sure and I couldn't find it just now, so all I can say about that is I think so. As for what license, that's a good question and I don't have a clue so I hope somebody who knows will answer that for you and for anyone reading this thread.

**Comment by dom96** [comment, marked as answer]
See https://github.com/modularml/mojo/issues/25
